type_templates/vrm.js

```diff
@@ -144,8 +259,27 @@ export default e => {
       localPlayer.setAvatarApp(app);
     };
 
+    app.updateQuality = async () => {
+      // const quality = getGfxSettingJSON('character').details;
+      // return await _setQuality(quality, app)
+      const quality = JSON.parse(localStorage.getItem('GfxSettings'));

```

Comment:
It's unacceptable for Totum to read local storage, since totum can run with environments that have no storage.

This must go through API.

Reply:
This is why I had get/set gfxSettings in the metaversefile, which you asked me to remove into a settings.js file.
https://github.com/webaverse/app/pull/2512#discussion_r823660325